rwatch/ui/layer/menu_layer.h

```diff
@@ -165,7 +165,11 @@ bool menu_layer_is_index_selected(const MenuLayer *menu_layer, MenuIndex *index)
 #endif
 
 #define MENU_CELL_BASIC_HEADER_HEIGHT ((const int16_t) 16)
-#define MENU_CELL_BASIC_CELL_HEIGHT ((const int16_t) DISPLAY_ROWS / 4)
+#ifdef PBL_RECT
+    #define MENU_CELL_BASIC_CELL_HEIGHT ((const int16_t) DISPLAY_ROWS / 4)
```

Comment:
One for later, but are we making assumptions on how big the menu is going to be (i.e full screen) and does this matter? Discuss

Reply:
I believe this is just an easy way to dynamically create a cell height. I'm pretty sure it can be changed in the menu implementation the dev uses.

Reply:
I was referring to DISPLAY_ROWS, which is an assumption that should stand up as long as everyone is consistant

Reply:
Using the division was a simple way to calculate the correct cell height on both platform. Nonetheless judging by how the CloudPebble emulator draws a `simple_menu_layer`, which of course uses `MENU_CELL_BASIC_CELL_HEIGHT` setting this on chalk to three items was not correct: [Screenshot](https://i.imgur.com/31MJ8Gt.png)
You can see, that there are four heights of basic cell visible on chalk and the original division by four resulted in an almost pixel-perfect result of `44px` (pixel-perfect would be `45px`).